src/composables/deepLinkApi.ts

```diff
@@ -12,12 +12,15 @@ import {
 } from '@/constants';
 import { useModals } from '@/composables/modals';
 
-export function useDeepLinkApi() {
-  const router = useIonRouter();
+let isDeepLinkUsed = false;
+
+export function useDeepLinkApi(doNotInitializeRouter?: boolean) {
```

Comment:
I suggest to pass the options as an object - same way as we do in other composables.